### PR TITLE
update make tasks

### PR DESCRIPTION
**Pull request by skeeey**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,81 +3,120 @@
...
-```
+## Deploy multicluster-controlplane in a cluster
 
-### Option 2: Start multicluster-controlplane with external etcd on Openshift Cluster 
+### Prepare
 
-#### Deploy etcd
-Set environmrnt variables and deploy etcd.
-* `ETCD_NS` (optional) is the namespace where the etcd is deployed in. The default is `multicluster-controlplane-etcd`.
+1. An Openshift Cluster or a kind cluster
```
Comment (clyang82) [inline README.md:16]: s/kind/KinD

File: hack/check/check-copyright.sh [hunk 2]
```diff
@@ -1,88 +0,0 @@
-#!/bin/bash
```
Comment (clyang82) [inline hack/check/check-copyright.sh:1]: why remove the check-copyright file? I remembered it is required in stolostron.

File: hack/deploy-etcd.sh [hunk 3]
```diff
@@ -1,86 +1,61 @@
...
-if [[ "${REUSE_CA}" != true ]]; then
-    if ! command -v go >/dev/null 2>&1; then
-        echo "Command go is not found"
-        exit 1
-    fi
+deploy_dir=${REPO_DIR}/_output/etcd/deploy
+cert_dir=${deploy_dir}/cert-etcd
 
-    if ! command -v $CFSSL >/dev/null 2>&1; then
-        echo "Command $CFSSL is not found, installing..."
```
Comment (clyang82) [inline hack/deploy-etcd.sh:32]: We need to check those commands before using. or include execute dependencies

**Review by clyang82 — COMMENTED** [review]
Overall looks good for me. 

Just provide some minor comments

File: Makefile [hunk 4]
```diff
@@ -2,101 +2,86 @@
...
 	go build -o bin/multicluster-controlplane cmd/main.go 
 .PHONY: build
 
 build-image:
 	docker build -f Dockerfile -t $(IMAGE_NAME) .
 .PHONY: build-image
 
-push-image:
-	docker push $(IMAGE_NAME)
-.PHONY: push-image
```
[the remark this answers is not in the document]
Comment (skeeey) [reply Makefile:37]: we don't need push-image task in our cicd

File: README.md (hunk 1 above)
Comment (skeeey) [reply README.md:16]: update all kind to KinD

File: hack/check/check-copyright.sh (hunk 2 above)
Comment (skeeey) [reply hack/check/check-copyright.sh:1]: the copyright is not required in the stolostron

File: hack/deploy-etcd.sh (hunk 3 above)
Comment (skeeey) [reply hack/deploy-etcd.sh:32]: done

File: hack/check/check-copyright.sh (hunk 2 above)
Comment (clyang82) [reply hack/check/check-copyright.sh:1]: I suggest to keep the copyright check. That is best practice to check it. If it does not work now, let us fix it in future.

Comment (skeeey) [reply hack/check/check-copyright.sh]: done

**Review by clyang82 — APPROVED** [review]
LGTM

**Pull request merged**